Cannot successfully run make docker

When I run `make docker` on ARM (Mac M2), I encounter the following error. Using Rancher


```bash
 > [exporter 2/6] RUN if [ "amd64" = "amd64" ]; then       microdnf update &&       microdnf install -y oracle-instantclient-release-el8 &&       microdnf install -y oracle-instantclient-basic &&       microdnf install glibc-2.28-251.0.2.el8_10.4     ; else       microdnf update &&       microdnf install wget libaio &&       wget https://download.oracle.com/otn_software/linux/instantclient/instantclient-basic-linux-arm64.rpm &&       rpm -ivh instantclient-basic-linux-arm64.rpm &&       ln -s /usr/lib/oracle/19.24 /usr/lib/oracle/21 &&       microdnf install glibc-2.28-251.0.2.el8_10.4     ; fi:
36.62 Running transaction test...
36.71 Updating: glibc-common;2.28-251.0.3.el8_10.25;aarch64;ol8_baseos_latest
36.82 Updating: glibc-minimal-langpack;2.28-251.0.3.el8_10.25;aarch64;ol8_baseos_latest
36.82 Updating: glibc;2.28-251.0.3.el8_10.25;aarch64;ol8_baseos_latest
36.95 Installing: glibc-gconv-extra;2.28-251.0.3.el8_10.25;aarch64;ol8_baseos_latest
37.14 Cleanup: glibc;2.28-251.0.3.el8_10.22;aarch64;installed
37.14 Cleanup: glibc-minimal-langpack;2.28-251.0.3.el8_10.22;aarch64;installed
37.15 Cleanup: glibc-common;2.28-251.0.3.el8_10.22;aarch64;installed
37.26 Complete.
38.44 error: No package matches 'oracle-instantclient-release-el8'
------
Dockerfile:44
--------------------
  43 |     # oracle-instantclient-release-23ai-el8 and paths below s/21/23/
  44 | >>> RUN if [ "$GOARCH" = "amd64" ]; then \
  45 | >>>       microdnf update && \
  46 | >>>       microdnf install -y oracle-instantclient-release-el8 && \
  47 | >>>       microdnf install -y oracle-instantclient-basic && \
  48 | >>>       microdnf install glibc-2.28-251.0.2.el8_10.4 \
  49 | >>>     ; else \
  50 | >>>       microdnf update && \
  51 | >>>       microdnf install wget libaio && \
  52 | >>>       wget https://download.oracle.com/otn_software/linux/instantclient/instantclient-basic-linux-arm64.rpm && \
  53 | >>>       rpm -ivh instantclient-basic-linux-arm64.rpm && \
  54 | >>>       ln -s /usr/lib/oracle/19.24 /usr/lib/oracle/21 && \
  55 | >>>       microdnf install glibc-2.28-251.0.2.el8_10.4 \
  56 | >>>     ; fi
  57 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c if [ \"$GOARCH\" = \"amd64\" ]; then       microdnf update &&       microdnf install -y oracle-instantclient-release-el8 &&       microdnf install -y oracle-instantclient-basic &&       microdnf install glibc-2.28-251.0.2.el8_10.4     ; else       microdnf update &&       microdnf install wget libaio &&       wget https://download.oracle.com/otn_software/linux/instantclient/instantclient-basic-linux-arm64.rpm &&       rpm -ivh instantclient-basic-linux-arm64.rpm &&       ln -s /usr/lib/oracle/19.24 /usr/lib/oracle/21 &&       microdnf install glibc-2.28-251.0.2.el8_10.4     ; fi" did not complete successfully: exit code: 1
make: *** [docker] Error 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -38,11 +38,10 @@ ENV GOARCH=${GOARCH:-amd64}
 RUN microdnf update && \
     microdnf install -y oracle-instantclient-release-23ai-el8-1.0-4.el8 && \
     microdnf install -y oracle-instantclient-basic-23.9.0.25.07-1.el8 && \
-    microdnf install -y glibc-2.28-251.0.3.el8_10.25 && \
-    ln -s /usr/lib/oracle/19.24 /usr/lib/oracle/23
+    microdnf install -y glibc-2.28-251.0.3.el8_10.25
 
-ENV LD_LIBRARY_PATH=/usr/lib/oracle/23/client64/lib:usr/lib/oracle/19.24/client64/lib
-ENV PATH=$PATH:/usr/lib/oracle/23/client64/bin:usr/lib/oracle/19.24/client64/bin
+ENV LD_LIBRARY_PATH=/usr/lib/oracle/23/client64/lib
+ENV PATH=$PATH:/usr/lib/oracle/23/client64/bin
 
 COPY --from=build /go/src/oracledb_exporter/oracle-db-appdev-monitoring /oracledb_exporter
 ADD ./default-metrics.toml /default-metrics.toml

--- a/Dockerfile
+++ b/Dockerfile
@@ -35,28 +35,14 @@ ENV DEBIAN_FRONTEND=noninteractive
 ARG GOARCH
 ENV GOARCH=${GOARCH:-amd64}
 
-# note: the 23ai arm drivers are not in yum yet, when they are can switch this to just use yum and not need
-# to wget the driver for arm.  also note that the permalink for otn download of drivers does not have version
-# in it, and does not appear to be a link vith version in it, so that link is very brittle and could break build
+RUN microdnf update && \
+    microdnf install -y oracle-instantclient-release-23ai-el8-1.0-4.el8 && \
+    microdnf install -y oracle-instantclient-basic-23.9.0.25.07-1.el8 && \
+    microdnf install -y glibc-2.28-251.0.3.el8_10.25 && \
+    ln -s /usr/lib/oracle/19.24 /usr/lib/oracle/23
 
-# second note: moved back to 21c drivers due to adb-s non-root connection issue. for 23ai, change rpm to
-# oracle-instantclient-release-23ai-el8 and paths below s/21/23/
-RUN if [ "$GOARCH" = "amd64" ]; then \
-      microdnf update && \
-      microdnf install -y oracle-instantclient-release-el8 && \
-      microdnf install -y oracle-instantclient-basic && \
-      microdnf install glibc-2.28-251.0.2.el8_10.4 \
-    ; else \
-      microdnf update && \
-      microdnf install wget libaio && \
-      wget https://download.oracle.com/otn_software/linux/instantclient/instantclient-basic-linux-arm64.rpm && \
-      rpm -ivh instantclient-basic-linux-arm64.rpm && \
-      ln -s /usr/lib/oracle/19.24 /usr/lib/oracle/21 && \
-      microdnf install glibc-2.28-251.0.2.el8_10.4 \
-    ; fi
-
-ENV LD_LIBRARY_PATH=/usr/lib/oracle/21/client64/lib:usr/lib/oracle/19.24/client64/lib
-ENV PATH=$PATH:/usr/lib/oracle/21/client64/bin:usr/lib/oracle/19.24/client64/bin
+ENV LD_LIBRARY_PATH=/usr/lib/oracle/23/client64/lib:usr/lib/oracle/19.24/client64/lib
+ENV PATH=$PATH:/usr/lib/oracle/23/client64/bin:usr/lib/oracle/19.24/client64/bin
 
 COPY --from=build /go/src/oracledb_exporter/oracle-db-appdev-monitoring /oracledb_exporter
 ADD ./default-metrics.toml /default-metrics.toml

